janusgraph-dist/pom.xml

```diff
@@ -13,6 +13,7 @@
 
     <properties>
         <top.level.basedir>${project.basedir}/..</top.level.basedir>
+        <cassandra-dist.version>4.0.3</cassandra-dist.version>
```

Comment:
Apart from the two different versions: If we now ship with Cassandra 4 by default, then this should definitely be named in the `CHANGELOG.md`. It currently doesn't even mention that JanusGraph is compatible with Cassandra 4.